apps/web/src/lang/fr.json

```diff
@@ -8151,5 +8151,113 @@
+    "description": "Description for checklist section in application education page."
+  },
+  "WkjRzF": {
+    "defaultMessage": "Lien vers votre curriculum vitæ",
+    "description": "Heading for checklist section in application education page."
+  },
+  "ZCUy93": {
+    "defaultMessage": "Erreur : La mise à jour de l’exigence relative aux études a échoué.",
+    "description": "Message displayed to user after education requirement fails to be updated."
+  },
+  "cT6KeA": {
+    "defaultMessage": "Veuillez choisir une option pour continuer.",
+    "description": "Alert message informing user to select an option first in application education page."
+  },
+  "d2s3n5": {
+    "defaultMessage": "Vous pouvez ajouter des expériences au moment <link>de créer une nouvelle expérience dans votre curriculum vitæ à l’étape précédente.</link>",
+    "description": "Deuxième message d’avertissement informant l’utilisatrice ou l’utilisateur qu’il doit ajouter une expérience à la page dédiée aux études de la demande d’emploi."
+  },
+  "fvE7Cx": {
+    "defaultMessage": "Apprentissage en cours d’emploi",
+    "description": "List item in applied work option in application education page."
+  },
+  "h0Mrq9": {
+    "defaultMessage": "<strong>technologie de l’information</strong>",
+    "description": "List item of experiences required in application education page."
+  },
+  "iMAe2n": {
+    "defaultMessage": "<strong>une autre spécialité pertinente à cette offre d’emploi</strong>",
+    "description": "List item of experiences required in application education page."
+  },
+  "lb2jFJ": {
+    "defaultMessage": "<strong>Il semble que vous n’ayez pas encore ajouté d’expérience à votre curriculum vitæ .</strong>",
+    "description": "Alert message informing user to add experience in application education page."
+  },
+  "qEYoGS": {
+    "defaultMessage": "Pour nous aider à comprendre comment vous répondez aux critères d’expérience et de formation minimaux, veuillez indiquer les options qui correspondent à votre situation, ainsi que les expériences de votre curriculum vitæ qui s’appliquent. Si les deux options s’appliquent à vous, tant mieux! N’hésitez pas à choisir l’option qui reflète le mieux vos qualifications.",
+    "description": "Description for radio group section in application education page."
+  },
+  "rGqohv": {
+    "defaultMessage": "Ma situation correspond à l’option de deux années d’études postsecondaires.",
+    "description": "Radio group option for education requirement filter in application education form."
+  },
+  "sHKNfX": {
+    "defaultMessage": "Vous pouvez ajouter des expériences propres à vos études en sélectionnant l’option \"Education and certificates\" (Études et certificats) au moment de <link>créer une nouvelle expérience pertinente à votre curriculum vitæ à l’étape précédente.</link>",
```

Comment:
@yonikid15 is there a reason why the English copy, Education and certificates, was used in French?